org.lflang/src/org/lflang/ast/ToLf.java

```diff
@@ -757,14 +760,26 @@ public MalleableString caseSerializer(Serializer object) {
   @Override
   public MalleableString caseKeyValuePairs(KeyValuePairs object) {
     // {KeyValuePairs} '{' (pairs+=KeyValuePair (',' (pairs+=KeyValuePair))* ','?)? '}'
-    if (object.getPairs().isEmpty()) return MalleableString.anyOf("");
+    if (object.getPairs().isEmpty()) {
+      return MalleableString.anyOf("");
+    }
     return new Builder()
         .append("{\n")
         .append(list(",\n", "", "\n", true, true, object.getPairs()).indent())
         .append("}")
         .get();
   }
 
+  @Override
+  public MalleableString caseBracedListExpression(BracedListExpression object) {
+    if (object.getItems().isEmpty()) {
+      return MalleableString.anyOf("{}");
+    }
+    // Note that this strips the trailing comma. There is no way
+    // to implement trailing commas with the current set of list() methods AFAIU.
+    return list(", ", "{ ", " }", false, false, object.getItems());
```

Comment:
Why insert the extra space after `{` and before `}`? This does not look idiomatic to me if I look at the examples.

Reply:
It looked nicer to me, but I don't care either way. If it's more idiomatic without the spaces then let's remove them